pkg/detector/detector.go

```diff
@@ -1144,14 +1144,14 @@ func (d *ResourceDetector) HandlePropagationPolicyDeletion(policyNS string, poli
 		// Must remove the labels from the resource template ahead of ResourceBinding, otherwise might lose the chance
 		// to do that in a retry loop(in particular, the label was successfully removed from ResourceBinding, but
 		// resource template not), since the ResourceBinding will not be listed again.
-		if err := d.CleanupLabels(binding.Spec.Resource, policyv1alpha1.PropagationPolicyNamespaceLabel, policyv1alpha1.PropagationPolicyNameLabel); err != nil {
+		if err := d.CleanupLabels(binding.Spec.Resource, policyv1alpha1.PropagationPolicyNamespaceLabel, policyv1alpha1.PropagationPolicyNameLabel, policyv1alpha1.PropagationPolicyPermanentIDLabel); err != nil {
```

Comment:
We should know that it's not a good design pattern to cleanup labels with a whitelist label, we may run into this same issue again when we introduce a new label.

Reply:
Another problem is that it is not easy to expand. If there are other cleaning operations, like clean annotations in the future, the changes will be relatively large. I'm working on it